packages/js/api-core-tests/data/order.js

```diff
@@ -24,7 +24,7 @@ const order = {
 };
 
 const productLineItems = {
-	name: '',
+	name: 'Handmade Wooden Gloves',
```

Comment:
Was this change needed?

Reply:
Oh, right! This change isn't needed anymore because it's already covered by the other search test (searching by `order_item_name`). I'll revert this change in my next commit.